openrr-client/src/clients/collision_check_client.rs

```diff
@@ -81,3 +81,47 @@ pub fn create_collision_check_client<P: AsRef<Path>>(
         )),
     )
 }
+
+#[cfg(test)]
```

Comment:
I want you to move the test code into `mod test` because you'll increase more tests. If the test is only one, it has no meaning actually.

```rust
mod test {
```

Reply:
You are right, I or another developer would add tests in the future.